app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,65 @@
  *= require_tree .
  *= require_self
  */
+
+@import url('https://fonts.googleapis.com/css2?family=Recursive:wght@300;400;600&display=swap');
+
```

Comment:
Holy CSS, you've got full stack in your future buddy 🤩

Reply:
Tried it out in rails server local host.  Love the font, buttons remind me of my favorite lego game from the 90s

Reply:
Is this calling a font API?? Bah gawd!